pkg/tests/tasks/security/certificate/yaml_configs.go

```diff
@@ -26,5 +26,93 @@ spec:
         type: PrivateKey
         privateKey:
           rootCADir: /etc/cacerts
+`
+	SelfSignedCa = `
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: istio-ca
+spec:
+  isCA: true
+  duration: 2160h # 90d
+  secretName: istio-ca
+  commonName: istio-ca
+  subject:
+    organizations:
+      - cluster.local
+      - cert-manager
+  issuerRef:
+    name: selfsigned
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: istio-ca
+spec:
+  ca:
+    secretName: istio-ca
+ `
+
+	certManagerSMCP = `
+kind: ServiceMeshControlPlane
+apiVersion: maistra.io/v2
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.3
+  tracing:
+    type: Jaeger
```

Comment:
We don't need tracing in this test.
```suggestion
    type: None
```

Reply:
updated to None